ui/values/dimensions.go

```diff
@@ -118,4 +118,6 @@ var (
 	MobileAppHeight = unit.Dp(780)
 
 	StartMobileView = unit.Dp(600)
+
+	Length1000 = 1000
```

Comment:
@ukane-philemon this should be the culprit 
```suggestion
	Length1000 = unit.Dp(1000)
```